treefmt.toml

```diff
@@ -0,0 +1,37 @@
+# One CLI to format the code tree - https://github.com/numtide/treefmt
+
+[formatter.nix]
+command = "./result/bin/alejandra"
```

Comment:
The other alternative is `$ cargo run -- .`

that one rebuilds but has a cache so it's faster

Reply:
That would lead to bad cycles during editing when `treefmt` is hooked with the editor, so we can't do this one, although it would have been a nice idea. I think those are luxury problems while working on an fmt.

I'll change it to `./target/debug/alejandra` & we can make sure that cargo build is triggered upon devshell entry